### DISCUSSION TITLE
Define the order of items in a nested collection

### DISCUSSION THREAD

**Discussion by spicy-sauce** [strapi/strapi#7815] [Legacy Topics]
Hi, 
Let's say I have an entity named "Employee" with a one to many relation to an entity named "Phones".
Now, I'd like to get a given employee by:
await strapi.query("employee").findOne({
        id: 5,
      });

Is there an option to sort the employee's phones by a given field in this table or should I do it once I have the phones in hand by some JS code?

Thanks in advance!

**Comment by derrickmehaffy** [comment]
Currently we don't have a way to define the order of relations (per this FAQ: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#relations-aren-t-maintaining-their-sort-order)

So you will need to add some kind of order field or work some JS magic to sort them.

**Reply by spicy-sauce** [reply]
I see, thanks for answering, really appreciated!
